containers/dotnet-postgres/.devcontainer/devcontainer.json

```
@@ -0,0 +1,40 @@
+{
+    "name": "C# (.NET) and PostgreSQL",
```

We've been including this in community contributed definitions.
```suggestion
    "name": "C# (.NET) and PostgreSQL (Community)",
```